gui/transmit_output.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+
+import serial
+
+# ser = serial.Serial("/dev/cu.usbserial-017543DC", 57600)
+# TODO: move functions here from robot and gui for proper categorization. Step 2 would be making sure data is not corrupt
```

Comment:
did u mean gui to robot? Seems like this file is for functions that will sned commands from GUI to robot.